<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GIS桌面软件-马骁</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/myRc.qrc">
    <normaloff>:/ico/qgis.ico</normaloff>:/ico/qgis.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpenVector"/>
    <addaction name="actionOpenRaster"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>测量</string>
    </property>
    <addaction name="actionArea"/>
    <addaction name="actionDistance"/>
   </widget>
   <widget class="QMenu" name="ConnectDataBase">
    <property name="title">
     <string>连接数据库</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actiongu"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="ConnectDataBase"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>1000</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_1"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenVector"/>
   <addaction name="actionOpenRaster"/>
   <addaction name="actionArea"/>
   <addaction name="actionDistance"/>
  </widget>
  <action name="actionOpenRaster">
   <property name="text">
    <string>打开栅格</string>
   </property>
  </action>
  <action name="actionOpenVector">
   <property name="text">
    <string>打开矢量</string>
   </property>
  </action>
  <action name="actionArea">
   <property name="text">
    <string>面积量算</string>
   </property>
  </action>
  <action name="actionDistance">
   <property name="text">
    <string>距离量算</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="actiongu">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/myRc.qrc"/>
 </resources>
 <connections/>
</ui>
